docs/Protocol/Glossary.md

```diff
@@ -2,12 +2,34 @@
 
```

Comment:
Please add "Invest Proposal", "Trader Pool", "Active portfolio" and "Validators"